src/garage/torch/algos/trpo.py

```diff
@@ -99,8 +98,17 @@ def _compute_objective(self, advantages, valids, obs, actions, rewards):
 
         return surrogate
 
-    def _optimize(self, itr, obs, actions, rewards, valids, baselines):
+    def _optimize(self, obs, actions, rewards, advantages):
+        """Performs a optimization.
+
+        Args:
+            obs (torch.Tensor): Observation from the environment.
+            actions (torch.Tensor): Actions fed to the environment.
+            rewards (torch.Tensor): Acquired rewards.
+            advantages (torch.Tensor): Advantage value at each step.
+
+        """
         self._optimizer.step(
-            f_loss=lambda: self._compute_loss(itr, obs, actions, rewards,
-                                              valids, baselines),
+            f_loss=lambda: -self._compute_objective(advantages, obs, actions,
```

Comment:
This is because I need to change the `_compute_loss` function for MAML.
(for MAML, it doesn't have advantage values but for `TRPO._optimize` function doesn't have baselines.